crates/loro-wasm/src/convert.rs

```diff
@@ -89,8 +89,15 @@ pub(crate) fn resolved_diff_to_js(value: &Diff, doc: &Arc<LoroDoc>) -> JsValue {
                 .unwrap();
             // set diff as array
             let arr = Array::new();
-            for (i, v) in list.iter().enumerate() {
-                arr.set(i as u32, delta_item_to_js(v.clone(), doc));
+            let mut i = 0;
+            for v in list.iter() {
+                let (a, b) = delta_item_to_js(v.clone(), doc);
+                arr.set(i as u32, a);
+                i += 1;
```

Comment:
Why not use `push`?

Reply:
There is no .push() method on this struct